client/src/pages/WeeklyTable.js

```diff
@@ -38,19 +40,28 @@ const getBookingsByRow = (bookings, rowsCount, week) => {
 const getBookingsByDesk = (bookings, week) => {
 	const bookingsByDayWithDesk = {};
 	week.forEach((day) => {
-		bookingsByDayWithDesk[day] = bookings.filter((booking) => booking.date === day && booking.desk !== null  );
+		bookingsByDayWithDesk[day] = bookings.filter(
+			(booking) => booking.date === day && booking.desk !== null
+		);
 	});
 
-	const bookingsByDesk = {}; 
-	desks.forEach((desk)=>{
-		bookingsByDesk[desk.name] = week.map((day) => bookingsByDayWithDesk[day].find((booking)=> booking.desk === desk.name));
+	const bookingsByDesk = {};
+	desks.forEach((desk) => {
+		bookingsByDesk[desk.name] = week.map((day) =>
+			bookingsByDayWithDesk[day].find((booking) => booking.desk === desk.name)
+		);
 	});
-		
+
 	return bookingsByDesk;
 };
 
+const getBookingByDay = (bookings, date) => {
```

Comment:
this would be better named countBookingsByDay

a function called getBooking(s)ByDay, I would expect to return all the bookings for the day